aries-site/src/examples/templates/filtering/FilteringWithRangeSelector.js

```diff
@@ -60,512 +48,47 @@ const allData = [
-              <Filters
-                data={data}
-                filtering={filtering}
-                setFiltering={setFiltering}
-                setData={setData}
-                filters={filters}
-                setFilters={setFilters}
-                filterData={filterData}
-                setSearch={setSearch}
-                // target is for demo purposes only, remove in production
-                target={containerRef && containerRef.current}
-              />
-            )}
-          </Box>
-          <RecordSummary data={data} filter={filtering} />
-        </Box>
-      </Header>
-      <Results data={data} />
-    </Box>
-  );
-};
-
-FilteringWithRangeSelector.propTypes = {
-  containerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
+// Define Data properties
+const properties = {
+  availability: { label: 'Availability' },
+  location: { label: 'Location' },
 };
 
-const Filters = ({
-  data,
-  filtering,
-  setData,
-  setFiltering,
-  setSearch,
-  target, // target is for demo purposes only, remove in production
-}) => {
-  const [availability, setAvailability] = useState(defaultAvailability);
-  const [filters, setFilters] = useState(defaultFilters);
-  const [location, setLocation] = useState(defaultLocation);
-  const [open, setOpen] = useState(false);
-  const [previousValues, setPreviousValues] = useState({});
-  const [showLayer, setShowLayer] = useState(false);
-  const size = useContext(ResponsiveContext);
-
-  useEffect(() => {
-    if (Object.keys(filters).length !== 0) {
-      setFiltering(true);
-    } else {
-      setFiltering(false);
-    }
-  }, [filters, setFiltering]);
-
-  const resetFilters = () => {
-    setAvailability(defaultAvailability);
-    setData(allData);
-    setFilters(defaultFilters);
-    setLocation(defaultLocation);
-    setSearch('');
-  };
-
-  const filterData = (array, criteria) => {
-    setFilters(criteria);
-    const filterKeys = Object.keys(criteria);
-    return array.filter(item =>
-      // validates all filter criteria
-      filterKeys.every(key => {
-        // ignores non-function predicates
-        if (typeof criteria[key] !== 'function') return true;
-        return criteria[key](item[key]);
-      }),
-    );
-  };
-
-  if (['xsmall', 'small'].includes(size)) {
-    return (
-      <>
-        <Box align="center" direction="row" gap="xsmall">
-          <Button
-            kind="toolbar"
-            icon={<Filter />}
-            alignSelf="start"
-            onClick={() => setShowLayer(true)}
-          />
-        </Box>
-        {showLayer && (
-          <Layer
-            onClickOutside={() => {
-              setShowLayer(!showLayer);
-              setLocation(previousValues.location);
-            }}
-            full
-            // target is for demo purposes only, remove in production
-            target={target}
+export const FilteringWithRangeSelector = () => (
+  <Page fill>
+    <PageContent>
+      <Box gap="medium">
```

Comment:
Similar/same comments as to whether all of these Boxes, fills, and heights are necessary.